### PR TITLE
CA-6912 #comment migrated lobbyist data_warehouse_job, result and log…

### PR DESCRIPTION
**Pull request by cjawale**
…s api code to github

### CODE REVIEW

File: lib/lobbyist/v2/data_warehouse_job.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module Lobbyist
+  module V2
+    class DataWarehouseJob < Lobbyist::V2::Base
+      attr_accessor :id, :company_id, :status,
+                    :params, :data_filename, :result_filename,
+                    :job_type, :completed_at
+
+      def self.update(id, params)
+        create_from_response(
+            put("/v2/data_warehouse_job/#{id}.json", params))
```
Comment (spiritbear) [inline lib/lobbyist/v2/data_warehouse_job.rb]: @cjawale convention is to use - (dash) in the URL

File: lib/lobbyist/v2/data_warehouse_job.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+module Lobbyist
+  module V2
+    class DataWarehouseJob < Lobbyist::V2::Base
```
Comment (cl-prefect) [inline lib/lobbyist/v2/data_warehouse_job.rb]: Style/Documentation: Missing top-level class documentation comment.

File: lib/lobbyist/v2/data_warehouse_job.rb (hunk 1 above)
Comment (cl-prefect) [inline lib/lobbyist/v2/data_warehouse_job.rb]: Layout/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.

Comment (cl-prefect) [inline lib/lobbyist/v2/data_warehouse_job.rb]: Layout/MultilineMethodCallBraceLayout: Closing method call brace must be on the line after the last argument when opening brace is on a separate line from the first argument.

File: lib/lobbyist/v2/data_warehouse_job.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+                    :params, :data_filename, :result_filename,
+                    :job_type, :completed_at
+
+      def self.update(id, params)
+        create_from_response(
+            put("/v2/data_warehouse_job/#{id}.json", params))
+      end
+    end
+  end
+end
```
Comment (cl-prefect) [inline lib/lobbyist/v2/data_warehouse_job.rb]: Layout/TrailingBlankLines: Final newline missing.

File: spec/lobbyist/v2/data_warehouse_job_spec.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+describe Lobbyist::V2::DataWarehouseJob do
+
```
Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Layout/EmptyLinesAroundBlockBody: Extra empty line detected at block body beginning.

File: spec/lobbyist/v2/data_warehouse_job_spec.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  # Test Case for Lobbist:V2::DataWarehouseJob Update.
+  it 'should update the DataWarehouseJob' do
+    VCR.use_cassette('v2/data_warehouse_job_update') do
+      dw_job = Lobbyist::V2::DataWarehouseJob.update(1, {status: "algorithm_completed", result_filename: 'CL-Test'})
```
Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Layout/SpaceInsideHashLiteralBraces: Space inside { missing.

Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.

Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.

Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Metrics/LineLength: Line is too long. [116/80]

Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Layout/SpaceInsideHashLiteralBraces: Space inside } missing.

File: spec/lobbyist/v2/data_warehouse_job_spec.rb [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+      expect(dw_job).to be_a(Lobbyist::V2::DataWarehouseJob)
+      expect(dw_job.status).to eq('algorithm_completed')
+    end
+  end
+
+  # Test Case for Lobbist:V2::DataWarehouseJob Update should raise error if id not present.
```
Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Metrics/LineLength: Line is too long. [91/80]

File: spec/lobbyist/v2/data_warehouse_job_spec.rb [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+  it 'should get the error if job is not present' do
+    VCR.use_cassette('v2/data_warehouse_job_update_error') do
+      expect{ Lobbyist::V2::DataWarehouseJob.update(11234, {status: "algorithm_completed",
```
Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Layout/SpaceBeforeBlockBraces: Space missing to the left of {.

Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks.

Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Layout/MultilineBlockLayout: Block body expression is on the same line as the block start.

Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Style/NumericLiterals: Use underscores(_) as decimal mark and separate every 3 digits with them.

Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Layout/SpaceInsideHashLiteralBraces: Space inside { missing.

Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.

Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.

Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Metrics/LineLength: Line is too long. [90/80]

File: spec/lobbyist/v2/data_warehouse_job_spec.rb [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+                                                            result_filename: 'CL-Test'}) }.to raise_error
```
Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Metrics/LineLength: Line is too long. [105/80]

Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Layout/SpaceInsideHashLiteralBraces: Space inside } missing.

Comment (cl-prefect) [inline spec/lobbyist/v2/data_warehouse_job_spec.rb]: Layout/BlockEndNewline: Expression at 18, 90 should be on its own line.

**Pull request closed without merging**